distribution/lib/Standard/Base/0.0.0-dev/src/Any.enso

```diff
@@ -78,7 +78,10 @@ type Any
                  a = 7 * 21
                  a == 147
     == : Any -> Boolean
-    == self that = if Meta.is_same_object self that then True else
+    == self that = self.equals_builtin that
+
+    old_equals : Any -> Boolean
+    old_equals self that = if Meta.is_same_object self that then True else
         self_meta = Meta.meta self
         that_meta = Meta.meta that
         case Pair.new self_meta that_meta of
```

Comment:
This probably should be dropped before merging.

Reply:
Definitely. I will clean it up.